docs/snapshot-single-delete/scenarios/_query_tree.py

```
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+import paramiko
+import sys
+
+HOST = "172.26.53.180"
+USER = "root"
+PWD = "admin@123"
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to: "PWD"

(S105)